docs/index.html

```diff
@@ -165,7 +165,7 @@ <h2>Other Resources</h2><p><a href="http://stackoverflow.com/questions/tagged/th
 <br class="clear">
 
 <footer>
-    Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc 3.5.5</a> on Fri Jun 15 2018 17:15:39 GMT-0700 (Mountain Standard Time) using the <a href="https://github.com/clenemt/docdash">docdash</a> theme.
+    Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc 3.5.5</a> on Thu Jul 19 2018 14:03:35 GMT+0200 (CEST) using the <a href="https://github.com/clenemt/docdash">docdash</a> theme.

```

Comment:
It would be great to see if we can ensure one standard time zone is generated for doc generation.